docker/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM rust:1.74.0-slim
+
+# Install python, needed for PythonScriptRouter
+RUN apt update && \
+    apt install -y python3 python3-pip libpython3-dev && \
+    rm -rf /var/lib/apt/lists/*
+
+# There are propably some users that want to use this libs in their PythonScriptRouter to e.g. make external HTTP calls
+RUN pip install --break-system-packages requests
+
+WORKDIR /trino-lb
+
+COPY Cargo.toml Cargo.toml
+COPY trino-lb trino-lb
+# TODO: Think about if we really want to ship trino-lb-bench
+COPY trino-lb-bench trino-lb-bench
+COPY trino-lb-core trino-lb-core
+COPY trino-lb-persistence trino-lb-persistence
+
+RUN cargo build --release --all-features
```

Comment:
I'd sligthly prefer the way we do this in our operator images with a build image and only copying the artifacts to the final image. That would remove the need to have rust in this image as well.
But no strong opinion here.

Reply:
Sure, totally missed that! Let me have a look

Reply:
Please have another look